pkg/datapath/loader/verifier_test.go

```diff
@@ -0,0 +1,457 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package loader
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"iter"
+	"log/slog"
+	"maps"
+	"os"
+	"path"
+	"slices"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/rlimit"
+	"github.com/cilium/hive/hivetest"
+
+	"github.com/cilium/cilium/pkg/bpf"
+	"github.com/cilium/cilium/pkg/defaults"
+	"github.com/cilium/cilium/pkg/lock"
+	"github.com/cilium/cilium/pkg/testutils"
+)
+
+var (
+	flagCiliumBasePath = flag.String("cilium-base-path", "", "Cilium checkout base path")
+	flagKernelVersion  = flag.String("kernel-version", kernelVersionNetNext.String(), "Kernel version to assume for verifier tests")
+	flagResultDir      = flag.String("result-dir", "", "Directory to write verifier complexity results and verifier logs to (temp directory if empty)")
+	flagFullLog        = flag.Bool("full-log", false, "Write full verifier log to file (default: false)")
+)
+
+func TestPrivilegedVerifier(t *testing.T) {
```

Comment:
Should this get a comment, explaining the test name schema? E.g. when seeing `TestPrivilegedVerifier/LXC/1/1`, the first number references the compile time config, and the second number the load time permutation?